scripts/generate_screenshots/apply_variant.sh

```
@@ -70,22 +70,28 @@ if [ "$expected_count" -eq 0 ]; then
 for lang_dir in "$VARIANT_DIR"/*/; do
   [ -d "$lang_dir" ] || continue
   lang=$(basename "$lang_dir")
-  actual_count=$(ls -1 "$lang_dir"*.png 2>/dev/null | wc -l | tr -d ' ')
-  if [ "$actual_count" -ne "$expected_count" ]; then
-    incomplete+="  - ${lang}: ${actual_count}/${expected_count} 枚"$'\n'
-  fi
+  for device in "${device_list[@]}"; do
+    device=$(echo "$device" | tr -d ' ')
+    [ -n "$device" ] || continue
+    display_type=$(get_display_type "$device")
+    actual_count=$(ls -1 "$lang_dir"*_"${display_type}"_*.png 2>/dev/null | wc -l | tr -d ' ')
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)